repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -75,6 +77,7 @@ import scala.io.Source
   */
 class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMode: ResponseMode, bodyMode: ResponseMode, desiredOutputStream: ServletOutputStream)
   extends javax.servlet.http.HttpServletResponseWrapper(originalResponse) with HeaderInteractor {
+  private val headerWarningLogger = LoggerFactory.getLogger(classOf[HttpServletResponseWrapper].getName + "_headerWarning")
```

Comment:
Should we change the name from `_headerWarning` to `.headerWarning`? Doing so makes `.` a consistent delimiter, and I can't come up with a reason why we need to use something different. I could just be missing it, though.

You could also use string interpolation here, but I don't think it particularly matters. That's probably a preference thing.